doc/src/contracts/connection_router.md

```diff
@@ -0,0 +1,139 @@
+# Connection Router Contract
+
+The connection router contract is responsible for routing messages to a chain and controlling gateways associated with a chain that is connected to Axelar.
+
```

Comment:
This would be a useful place to add some comments about the governance and admin accounts (what each of them can do).